_posts/2024-06-10-first-cockpit-files-release.md

```diff
@@ -0,0 +1,27 @@
+---
+title: First Cockpit-Files release
+author: jelle
+date: 2024-06-10
+category: release
+tags: cockpit files manager browser
+---
+
+Last Friday we made the first release of cockpit-files, a new plugin for Cockpit to manage files on your server. Development on this new plugin was started by our GSoC student [Mahmoud Hamdy](https://github.com/MahmoudHamdy02) with the intention to replace existing [cockpit-navigator](https://github.com/45drives/cockpit-navigator/) plugin from 45Drives with a modern, automated tested and Cockpit styled plugin.
```

Comment:
Cockpit-styled with a hyphen.  But maybe mention Patternfly?  Is that a thing we do?  I notice that it's mentioned a lot when talking about the Anaconda webui...